### ISSUE TITLE
Add the Common Voice Single Word Dataset

### ISSUE THREAD

**Issue by StuartIanNaylor** [wenet-e2e/wekws#57]
https://commonvoice.mozilla.org/en/datasets

Finding single word datasets for English is hard and the Single Word Dataset from Common Voice is a rarity being multi national.
Also it has a very useful sample 'Hey' that can be concatenated with other keywords.
I used Sox to arrange by pitch/trim and then used Hey from Common Voice and Marvin from the Google Dataset to concatenate 'Hey Marvin' to create a good phonetically unique keyword.

**Comment by robin1001** [comment]
Does it work for real hotword system?

**Comment by StuartIanNaylor** [comment]
Not sure what you mean by 'real' hotword system as yes if you curate the dataset then you can create if if still slightly sparse for qty of KW.
Common voice is now part of ML Commons which is also single word and provides far more non-kw but also something I have been playing with due to the binary nature of kw & not-kw is using additional 'sounds like' classification that also can be constructed in a similar way that helps massively with false positives, that gives more cross entropy for the training to work on.
Also separate a noise classification and voice only not-kw can help much due to the way softmax can work where a simple binary kws can return a high softmax on the KW not because its a good fit but that its a very bad fit to !kw.
